src/api/error.ts

```diff
@@ -112,8 +152,8 @@ async function handleError( e: Object, options: Object = {} ): Object {
   let errorJson;
   try {
     errorJson = await e.response.json( );
-  } catch ( jsonError ) {
-    if ( jsonError.message.match( /JSON Parse error/ ) ) {
+  } catch ( jsonError: unknown ) {
+    if ( jsonError instanceof Error && jsonError.message.match( /JSON Parse error/ ) ) {
```

Comment:
Functional change 2: If a non `Error` is raised, then `jsonError.message` won't exist

Reply:
Sounds good, thanks for the callout.